tests/src/JIT/Regression/JitBlue/GitHub_23530/GitHub_23530.cs

```diff
@@ -36,10 +36,13 @@ static unsafe float fmaTest()
 
         static int Main(string[] args)
         {
-            float result = fmaTest();
-            if (Math.Abs(result - 5.0F) > System.Single.Epsilon)
+            if (Fma.IsSupported)
             {
-                return -1;
+                float result = fmaTest();
+                if (Math.Abs(result - 5.0F) > System.Single.Epsilon)
```

Comment:
I would say we can compare this against `Math.FusedMultiplyAdd` now; but that will get accelerated if `Fma.IsSupported` returns true

Reply:
I'd just as soon be explicit here

Reply:
If you are wanting to be explicit, then this result should be deterministic and you shouldn't need anything more than `result != 5.0f`.

If this isn't deterministic, then we have other problems 😄